src/App.js

```diff
@@ -1,42 +1,31 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import pressButton from './redux/actions';
 import logo from './logo.svg';
 import './App.css';
 import Button from './button';
 import Display from './display';
 import { BUTTON_PROPERTIES_ARRAY } from './constants';
-import { updateResult } from './pressButtonLogic';
 import './calculatorStyle.css';
+import { Provider } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import PropTypes from 'prop-types';
 
 class App extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            result: ['0'],
-            operators: {
-                array: [],
-                redundant: 0,
-            },
-            default: true,
-            point: false,
-        };
-        this.updateState = this.updateState.bind(this);
-    }
-
-    updateState(value) {
-        this.setState({ ...updateResult(value, this.state) });
+    handleUpdateState(value) {
+        this.props.pressButton(value);
     }
 
     getButtonStyle(properties, index) {
+        const { defaultValue, operators } = { ...this.props };
         let newProperties = { ...BUTTON_PROPERTIES_ARRAY[index] };
-        if (index === 0) {
-            if (
-                this.state.default === false ||
-                this.state.operators.array.length > 0
-            ) {
-                newProperties.value = 'C';
-            } else {
-                newProperties.value = 'AC';
-            }
+        if (index !== 0) {
```

Comment:
Don't be afraid to add an empty line after you define the variables. It makes the code easier to read.